Packs/Campaign/Scripts/SplitCampaignContext/SplitCampaignContext.yml

```
@@ -0,0 +1,64 @@
+commonfields:
+  id: SplitCampaignContext
+  version: -1
+name: SplitCampaignContext
+script: ''
+type: python
+comment: Splits incident with similarity in context data to below and above threshold.
+tags: []
+enabled: true
+args:
+- name: SimilarityThresholdToSplitBy
```

```suggestion
- name: similarity_threshold
```
don't forget to regenerate the readme after applying this